docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -36,16 +27,102 @@ Here is an example of sending an access request via Teleport's Slack plugin:
 
 ## Prerequisites
 
-- Slack admin privileges to create an app and install it to your workspace
-
 (!/docs/pages/includes/commercial-prereqs-tabs.mdx!)
 
+- Slack admin privileges to create an app and install it to your workspace. Your
+  Slack profile must have the "Workspace Owner" or "Workspace Admin" banner
+  below your profile picture.
+
 (!/docs/pages/includes/tctl.mdx!)
 
-## Step 1/7. Install the Teleport Slack plugin
+## Step 1/8. Define RBAC resources
+
+Before you set up the Slack plugin, you will need to enable Access Requests in
+your Teleport cluster. For the purpose of this guide, we will define an
+`editor-requester` role, which can request the built-in `editor` role, and
+an `editor-reviewer` role that can review requests for the `editor` role. 
 
-We currently only provide `linux-amd64` binaries. You can also compile these plugins
-from source.
+Create a file called `editor-request-rbac.yaml` with the following content:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: editor-reviewer
+spec:
+  allow:
+    review_requests:
+      roles: ['editor']
+---
+kind: role
+version: v5
+metadata:
+  name: editor-requester
+spec:
+  allow:
+    request:
+      roles: ['editor']
+      thresholds:
+        - approve: 1
+          deny: 1
+```
+
+Create the roles  you defined:
+
+```code
+$ tctl create -f editor-request-rbac.yaml
+role 'editor-reviewer' has been created
+role 'editor-requester' has been created
+```
+
+Allow yourself to review requests by users with the `editor-requester` role by
+assigning yourself the `editor-reviewer` role. First, retrieve your user
+definition:
+
+```code
+$ TELEPORT_USER=$(tsh status | awk -F "      " '/Logged in as/{print $2}')
+$ tctl get user/${TELEPORT_USER?} > user.yaml
+```
+
+Edit `user.yaml` to add the `editor-reviewer` role:
+
+```diff
+ spec:
+   roles:
+   - access
+   - editor
++  - editor-reviewer
+```
+
+Update your user definition:
+
+```code
+$ tctl create -f user.yaml
+```
+
+Log out of Teleport and log in again. You will now have the ability to review
+requests for the `editor` role.
+
+Create a user called `myuser` who has the `editor-requester` role as well as the
+built-in `access` role and the `ubuntu` login. This user cannot edit your
+cluster configuration unless they request the `editor` role:
+
+```code
+$ tctl users add myuser --roles=access,editor-requester --logins=ubuntu
```

Comment:
Any need for the `access` role and `ubuntu` login? If someone follows this guide on their prod cluster, they will be creating a local user with really wide access, probably best to avoid

Reply:
Good point—I'll remove the extraneous permissions